packages/custom-field/src/vaadin-custom-field-mixin.js

```diff
@@ -181,9 +181,10 @@ export const CustomFieldMixin = (superClass) =>
      * @return {boolean}
      */
     checkValidity() {
-      const invalidFields = this.inputs.filter((input) => !(input.validate || input.checkValidity).call(input));
-
-      if (invalidFields.length || (this.required && !this.value.trim())) {
+      if (
```

Comment:
Looks like this is the wrong place to fix the problem completely.

The same thing happens as well in other methods, e.g. here:
```
_shouldRemoveFocus(event) {
      const { relatedTarget } = event;
      return !this.inputs.some((el) => relatedTarget === (el.focusElement || el));
    }
```
![image](https://github.com/user-attachments/assets/2d1b18ed-5eb2-403d-99c7-3186345fa581)

Reply:
Good point, thanks. I will check other methods too.

Reply:
Updated the PR. I wasn't able to reproduce this exact error though as typically the web component can only receive and then lose focus when it is connected to the DOM and by this point of time `inputs` is set to empty array.